doc/content/examples/fuel_cycle/plot_comparison.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
+import numpy as np
+import scipy.constants as scc
```

Comment:
Importing `scipy` just for converting from seconds to days seems a bit unecessary imo

Reply:
Yeah, but if it's all the same to you, I'd like to leave that one. It's readable and not very susceptible to error by typo. `Pandas` and `matplotlib` are much heavier than `scipy.constants` and I don't know anybody who doesn't have `scipy` on their machine.

Reply:
Sure